packages/api/tests/lambdas/sf-event-sqs-to-db-records/test-index.js

```diff
@@ -411,3 +413,31 @@ test('Lambda sends message to DLQ when writeRecords() throws an error', async (t
 
   t.is(handlerResponse[0][1].body, sqsEvent.Records[0].body);
 });
+
+test('writeRecords() discards an out of order message that is older than an existing message without error or write', async (t) => {
+  const {
+    cumulusMessage,
+    granuleModel,
+    pdrModel,
+    knex,
+    pdrName,
+    granuleId,
+  } = t.context;
+
+  const pdrPgModel = new PdrPgModel();
+  const granulePgModel = new GranulePgModel();
+
+  cumulusMessage.meta.status = 'completed';
```

Comment:
So the test setup is "out of order" because the "completed" message is written before the "running" one. And we're just proving that an error is not thrown?

What is guaranteeing that the first message is "older" than the second one? Is that guaranteed? Is it important for this test?

Reply:
I will add a second unit test that tests the createdAt condition.

Reply:
https://github.com/nasa/cumulus/pull/2156/commits/1027bd6a4454d070ed340d950ee02e03372f90d2